test/end-to-end/config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "root": "http://localhost:3000/",
+    "api": {
+        "uri": "http://localhost:4000/",
```

Comment:
`API_URL` is what the docker `entrypoint.sh` uses so it should do the same, or pick up changes from public/js/config.js (remembering to check for `welderApiRelative`